frontend/js/helpers/config.js.example

```diff
@@ -10,6 +10,24 @@ METRICS_URL = "__METRICS_URL__"
 METRIC_MAPPINGS = {
 
 
+    'disk_used_statvfs_system': {
+        'clean_name': 'Disk Usage',
+        'source': 'statvfs syscall',
+        'explanation': 'Disk used space via statvfs syscall',
```

Comment:
I would make it clearer here that it is the host system and not a container.

Reply:
thx